apps/studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnType.tsx

```diff
@@ -67,8 +67,11 @@ const ColumnType = ({
   onOptionSelect = noop,
 }: ColumnTypeProps) => {
   const [open, setOpen] = useState(false)
-  // @ts-ignore
-  const availableTypes = POSTGRES_DATA_TYPES.concat(enumTypes.map((type) => type.format))
+  const availableTypes = POSTGRES_DATA_TYPES.concat(
+    enumTypes.map((type) =>
+      type.schema === 'public' ? type.format.replaceAll('"', '') : type.format
+    )
+  )
   const isAvailableType = value ? availableTypes.includes(value) : true
```

Comment:
Just to confirm, this `value` doesn't need a `.replaceAll('"', '')` as well?

Reply:
value here should be okay since its a passed prop
- When selecting a new column type, we'd "sanitize" it when we selected the option, so value here will have the double quotes stripped already if needed
- When viewing a column's type though, i noticed that if the enum is not from the public schema, the user will not be able to edit the column, but i feel like this needs to be fixed on pg-meta's end? it seems like the type of the column does not contain the type's schema
![image](https://github.com/user-attachments/assets/a332a9b2-0921-4798-8f0c-c489815973cc)

Reply:
^ this bug is an existing issue though, so probably can address separately

Reply:
hmm one sec - i noticed this can be fixed from dashboard, not pgmeta

Reply:
okay updated this, should be fixed now 😄